awscli/customizations/paginate.py

```diff
@@ -73,6 +73,32 @@ def get_paginator_config(session, service_name, operation_name):
     return operation_paginator_config
 
 
+def add_paging_description(help_command, **kwargs):
+    if not isinstance(help_command.obj, model.OperationModel):
```

Comment:
It would be good to put a comment here saying that this customization should only be applied to operations and that is the reason you are checking to see if the ``help_command.obj`` is a ``model.OperationModel``.